tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/ProjectSystem/ActiveConfiguredProjectsLoaderTests.cs

```diff
@@ -15,12 +15,13 @@ public class ActiveConfiguredProjectsLoaderTests
         public async Task WhenActiveConfigurationChanges_LoadsConfiguredProject(string[] configurationNames)
         {
             var configurationGroups = IConfigurationGroupFactory.CreateFromConfigurationNames(configurationNames);
+            var configuredProject = ConfiguredProjectFactory.Create();
 
             var results = new List<string>();
             var project = UnconfiguredProjectFactory.ImplementLoadConfiguredProjectAsync(configuration =>
             {
                 results.Add(configuration.Name);
-                return Task.FromResult<ConfiguredProject?>(null);
+                return Task.FromResult(configuredProject);
```

Comment:
Behaviour change. The `ConfiguredProject` should not be `null`. Ditto several times below.